app/controllers/principal_roles_controller.rb

```diff
@@ -144,4 +144,9 @@ def respond_to_destroy principal_role, user, global_roles
       end
     end
   end
+
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.